app.json

```diff
@@ -1,56 +0,0 @@
-{
```

Comment:
I started off by just removing `WHEELMAP_TOKEN` but I think this file is not actually doing anything useful. We certainly haven't been keeping it up to date.
By the looks of it, we put this in place to support Heroku review apps, which we don't use anymore.